src/python/pants/engine/internals/graph.py

```diff
@@ -661,6 +699,12 @@ async def resolve_dependencies(
         subproject_roots=global_options.options.subproject_roots,
     )
 
+    # If this is a base target, inject dependencies on its subtargets.
+    subtarget_addresses: Tuple[Address, ...] = ()
+    if request.field.address.is_base_target:
+        subtargets = await Get(Subtargets, Address, request.field.address)
+        subtarget_addresses = tuple(t.address for t in subtargets.subtargets)
```

Comment:
Given this, I'm confused why more of the files aren't showing up?

```
▶ ./pants dependencies src/python/pants/util:util

3rdparty/python:ansicolors
3rdparty/python:dataclasses
3rdparty/python:typing-extensions
src/python/pants/__init__.py:init
src/python/pants/util/__init__.py
src/python/pants/util/dirutil.py
src/python/pants/util/meta.py
src/python/pants/util/strutil.py
```

Reply:
Veeeery good question, heh.

Reply:
Ah. Because:
```diff
     else:
-        target_roots = await Get(Targets, Addresses, addresses)
+        target_roots = await Get(UnexpandedTargets, Addresses, addresses)
         dependencies_per_target_root = await MultiGet(
             Get(Targets, DependenciesRequest(tgt.get(DependenciesField))) for tgt in target_roots
         )
```

Reply:
...and. Now that I think about it. Is the injection of subtargets in `DependenciesRequest` completely redundant? You get out `Addresses`, and the expansion to `Targets` will handle it...? Will remove.